tools/build.hpp

```diff
@@ -80,6 +81,22 @@ struct build_project {
    }
 
    int32_t exec() {
+      if (std::system("which cdt-cpp > /dev/null 2>&1") != 0 ){
+          system::error_log("cannot build the project, because CDT is not installed properly in your system!\n"
+                           "See https://github.com/AntelopeIO/cdt for the details of installing CDT");
+          return -1;
+      }
+      else {
+         // //'man cmake' indicates cmake --find-package is kept for compatibility but should not be used in new projects.
+         // // If cdt.pc is added in CDT in the future for tool pkg-config, 'pkg-config --exists cdt' can be used here
+         //
+         // if (std::system("cmake --find-package -DNAME=cdt -DCOMPILER_ID=GNU -DLANGUAGE=C -DMODE=EXIST > /dev/null 2>&1") !=0){
+         //    system::error_log("cannot build the project, CDT is not installed properly, cmake find_package(cdt) failed!\n"
+         //                   "Make sure cdt-cpp and related *.cmake files are properly installed in your system!");
+         //    return -2;
+         // }
```

Comment:
Lets reduce the cruft. A shorter comment above the if statement is more than adequate. Something like:
`'cmake --find-package' is deprecated, so use 'cdt-cpp --version' instead.` 
Or whatever command we settle on.